desc/objectives/linear_objectives.py

```diff
@@ -3238,3 +3234,393 @@ def compute(self, params, constants=None):
 
         """
         return params["Psi"]
+
+
+class FixNearAxisR(_FixedObjective):
+    """Fixes an equilibrium's near-axis behavior in R to specified order.
+
+        NOTE: does not also fix the axis R , for that
+        use the FixAxisR objectives.
+
+    Parameters
+    ----------
+    eq : Equilibrium
+        Equilibrium that will be optimized to satisfy the Objective.
+    nae_eq : Qsc, optonal
+        pyQSC Qsc object describing the NAE solution to fix the equilibrium's
+        near-axis behavior to. If None, will fix the equilibrium's current near
+        axis behavior.
+    order : int
```

Comment:
I feel like if order is 0 this should fix the axis? So that for most cases you only need this not this + fixed axis

Reply:
also if order can only be a discrete value it should be noted like 
`order : {0, 1, 2}`

Reply:
Yea I can do that but I just need to rearrange some of the logic in the utility functions